### DISCUSSION TITLE
Energy Dashboard in Home Assistant doesnt seem to include Non-Essential Load?

### DISCUSSION THREAD

**Discussion by emjayelle01** [kellerza/sunsynk#105] [General]
Hi,

I've recently installed my Sunsynk 8KW with RS485 cabled connection. 
First of all, thank you for all the work done!
I have managed to get the data to appear in Home Assistant and have been setting up my various dashboards.
One of the things I've noticed is the Energy Dashboard in Home Assistant doesn't appear to include the Non-Essential Load? My geysers and swimming pool are on non-essential and their respective usages for this morning aren't  showing up on the Usage, Distribution or Sources section.
Am I missing something? 
Note that I have been able to separate them on a custom power distribution card. I'm just trying to get the Energy Tab to correlate. 

Many thanks

**Comment by kellerza** [comment]
You are most likely using the wrong sensors, but it might also depend on your inverter installation.

**Reply by emjayelle01** [reply]
Hi,

I've changed the sensor I'm using to 'SS Day Grid Import' - would this be the correct one?

**Reply by kellerza** [reply]
Please try the updated docs: https://kellerza.github.io/sunsynk/guide/energy-management
